.github/workflows/actionlint.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         id: setup-homebrew
         uses: Homebrew/actions/setup-homebrew@master
         with:
+          core: true
```

Comment:
Don't think we need core here. No `HOMEBREW_NO_INSTALL_FROM_API` used and this is Linux too.